File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio22/Java/ColumnasVector.java
import java.util.Scanner;

public class ColumnasVector {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Ingrese el número de filas de la matriz: ");
        int filas = scanner.nextInt();

        System.out.print("Ingrese el número de columnas de la matriz: ");
        int columnas = scanner.nextInt();

        int[][] matriz = new int[filas][columnas];
        for (int i = 0; i < filas; i++) {
            System.out.print("Ingrese los elementos de la fila: ");
            for (int j = 0; j < columnas; j++) {
                matriz[i][j] = scanner.nextInt();
            }
        }

        int[] vectorSumaColumnas = new int[columnas];
        for (int j = 0; j < columnas; j++) {
            for (int i = 0; i < filas; i++) {
                vectorSumaColumnas[j] += matriz[i][j];
            }
        }

        System.out.print("Vector suma de columnas: ");
        for (int elemento : vectorSumaColumnas) {
            System.out.print(elemento + " ");
        }
    }
}

File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio22/Python/ColumnasVector.py
filas = int(input("Ingrese el número de filas de la matriz: "))
columnas = int(input("Ingrese el número de columnas de la matriz: "))

matriz = []
for _ in range(filas):
    fila = list(map(int, input("Ingrese los elementos de la fila: ").split()))
    matriz.append(fila)

vectorSumaColumnas = [0] * columnas

for fila in matriz:
    for j in range(columnas):
        vectorSumaColumnas[j] += fila[j]

print("Vector suma de columnas:", vectorSumaColumnas)